src/content/docs/en/recipes/dynamically-importing-images.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Dynamically Import Images
+description: Learn how to dynamically import images using Vite's import.meta.glob function
+i18nReady: true
+type: recipe
+---
+import FileTree from '~/components/FileTree.astro'
+
+When working with custom components that include images, there will be times where you will need to dynamically import the image paths. But if the images are located in the `src` directory, it is possible to run into Astro or Vite import errors.  
+
+In this recipe, you will learn how to dynamically import your images the correct way using Vite's `import.meta.glob` function. 
+
+## Recipe
+
+1. Create a new `assets` folder under the `src` directory and add your images inside that new folder. 
```

Comment:
Maybe we can add a notice about the `assets` folder here : it's just a convention but people are free to put images wherever they want. They'll just need to update the string the `glob` function

Reply:
I added the note 👍